File: Repositorio fullstack U.Distrital/Modulo 1 Fundamentos de programacio II/punto1.php
<?php
$AS = $_POST["AS"];
$CAL = $_POST["CAL"];
$APROBACION = 80;

if ($AS >= $APROBACION) {
    if ($CAL >= 70) {
        echo "Aprobado";
    } else {
        echo "Reprobado";
    }
} else {
    echo "Reprobado";
}
?>

File: Repositorio fullstack U.Distrital/Modulo 1 Fundamentos de programacio II/punto1.py
AS = int(input("Ingrese el porcentaje de asistencias: "))
CAL = int(input("Ingrese la calificación: "))
APROBACION = 80

if AS >= APROBACION:
    if CAL >= 70:
        print("Aprobado")
    else:
        print("Reprobado")
else:
    print("Reprobado")
